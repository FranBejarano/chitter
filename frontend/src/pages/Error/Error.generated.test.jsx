import React from "react";
import renderer from 'react-test-renderer';
import Error from "./Error";

jest.mock("react-router-dom");

const renderTree = tree => renderer.create(tree);
describe('<Error>', () => {
  it('should render component', () => {
    expect(renderTree(<Error
    />).toJSON()).toMatchSnapshot();
  });

});