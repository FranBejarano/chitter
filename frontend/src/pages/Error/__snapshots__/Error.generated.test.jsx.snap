// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error> should render component 1`] = `
<div
  className="text-center my-8 space-y-5"
>
  <h2
    className="font-bold text-4xl"
  >
    Error, page not found
  </h2>
  <p
    className="pb-2"
  >
    Please go back to login 
    <br />
  </p>
</div>
`;
